types/canvasjs/canvasjs-tests.ts

```diff
@@ -1,248 +1,246 @@
+import * as CanvasJS from "canvasjs";
```

Comment:
@sheetalkamat That documentation assumes some kind of automatic replacement of the default export a la what `--allowSyntheticDefaultExports` allows. If you look at their [code](https://github.com/Tsur/canvasjs/blob/master/src/main/index.js), it is exported to be used as a namespace like this.